parsec/api/protocol/base.py

```diff
@@ -80,13 +80,23 @@ class RequireGreaterTimestampRepSchema(BaseRepSchema):
 
 
 class TimestampOutOfBallparkRepSchema(BaseRepSchema):
+    """This schema has been added to API version 2.4.
```

Comment:
We should add a `TODO: This backward compatibility should be removed once Parsec<2.4 support is dropped`

Reply:
:+1:

Reply:
3eab1f05ffae0aa8d3997b1b95dea3b113f00a3d